pr-data.csv

```diff
@@ -744,7 +744,7 @@ https://github.com/alibaba/fastjson2,450d9fe5a39f9f911f06882a302a585b93a586fa,fa
 https://github.com/alibaba/fastjson2,450d9fe5a39f9f911f06882a302a585b93a586fa,fastjson1-compatible,com.alibaba.fastjson.jsonp.JSONPParseTest1.test_f,ID,Accepted,https://github.com/alibaba/fastjson2/pull/2034,https://github.com/alibaba/fastjson2/issues/2033
 https://github.com/alibaba/fastjson2,3de60a9769f2c95cbceef13b0158b1af934ab323,core,com.alibaba.fastjson2.codec.JSONBTableTest3.test_0,ID,Opened,https://github.com/alibaba/fastjson2/pull/3097,https://github.com/alibaba/fastjson2/issues/3096
 https://github.com/alibaba/fastjson2,3de60a9769f2c95cbceef13b0158b1af934ab323,core,com.alibaba.fastjson2.codec.JSONBTableTest3.test_1,ID,Opened,https://github.com/alibaba/fastjson2/pull/3097,https://github.com/alibaba/fastjson2/issues/3096
-https://github.com/alibaba/fastjson2,450d9fe5a39f9f911f06882a302a585b93a586fa,core,com.alibaba.fastjson2.dubbo.DubboTest4.test6,ID,,,
+https://github.com/alibaba/fastjson2,450d9fe5a39f9f911f06882a302a585b93a586fa,core,com.alibaba.fastjson2.dubbo.DubboTest4.test6,ID,Opened,https://github.com/alibaba/fastjson2/pull/3150,https://github.com/alibaba/fastjson2/issues/3149
```

Comment:
You have fastjson2/pull/3150 as both `Opened` and `Accepted` in the same PR here.

Reply:
Sorry about that, updated to accepted